rusk-wallet/src/bin/command.rs

```diff
@@ -429,7 +454,7 @@ pub(crate) enum Command {
         amt: Dusk,
 
         /// Max amount of gas for this transaction
-        #[clap(short = 'l', long, default_value_t = DEFAULT_STAKE_GAS_LIMIT)]
+        #[clap(short = 'l', long, default_value_t = DEFAULT_LIMIT_STAKE)]
```

Comment:
as above, stake-related constant shouldn't be used here

Reply:
same

Reply:
and here too